examples/pyfar_demo.ipynb

```diff
@@ -174,8 +175,7 @@
    "outputs": [],
    "source": [
     "time_data = x_power.time\n",
-    "freq_data_normalized = x_power.freq\n",
-    "freq_data_raw = x_power.freq_raw"
+    "freq_data = x_power.freq"
```

Comment:
It looks as if, we lost changes here red lines 177 and 178 are new and green line 178 is outdated

Reply:
thanks for noting, changed it